test/graphgym/example_node.yml

```diff
@@ -11,8 +11,10 @@ dataset:
   edge_encoder_name: Bond
 train:
   batch_size: 128
-  eval_period: 1
+  eval_period: 2
```

Comment:
If this is change is not needed, please keep the default configs unchanged, so that users can reproduce the examples in older versions of PyG GraphGym.

Reply:
This is modified to see if the the ``cfg.train.skip_train_eval`` option is functioning correctly. If ``cfg.train.eval_period`` is set to 1, there's no difference regardless of whether ``skip_train_eval`` is ``True`` or ``False``.

I could also create a new copy of this yml and keep the ``eval_peirod`` in the original yml to be 1 for the purpose of reproducing older examples. Let me know if you prefer this option.

Reply:
I see. This makes sense. I think having two examples may not be necessary, so you can go ahead with this change